### ISSUE TITLE
About the training code

### ISSUE THREAD

**Issue by YeFengyu819** [multimodallearning/convexAdam#17]
Hello,
Thank you for the great work and sharing the code!

I am sorry to ask where is the training code, if it is shared. By the way, I find there is a optimizer.step() for the validation set in convex_adam_MIND.py, I am not sure if it is the training code.

Thankyou
Best wishes

**Comment by mattiaspaul** [comment]
Hi YeFengyu819,
thanks for your question. There is no training of network parameters within this repository - the only deep learning used would be the nnUNet when employing semantic features. There is the hyperparameter optimisation described in our new TMI paper in more detail, but otherwise the method is mainly optimisation-driven. Convex+Adam MIND requires gradients for a classic optimisation on the test pairs with MIND similarity loss.
Best wishes, Mattias
